tests/Server/ServerTestCase.php

```
@@ -0,0 +1,217 @@
+        $process = proc_open($command, $descriptors, $pipes);
+        if (!is_resource($process)) {
+            self::fail('Failed to start PHP built-in server');
+        }
+
+        $status = proc_get_status($process);
+        self::$serverPid = $status['pid'];
+
+        // Wait for server to be ready (up to 3 seconds)
+        $ready = false;
+        for ($i = 0; $i < 30; $i++) {
+            $sock = @fsockopen(self::$host, self::$port, $errno, $errstr, 0.1);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$errno'. (undefined)

(UnusedLocalVariable)